pkg/controller/testsuite/testsuite_controller.go

```diff
@@ -210,6 +212,16 @@ func (r *ReconcileTestSuite) ensureStatusIsUpToDate(ctx context.Context, suite t
 	return r.statusService.EnsureStatusIsUpToDate(suite, pods)
 }
 
+func (r *ReconcileTestSuite) setErrorStatus(ctx context.Context, suite *testingv1alpha1.ClusterTestSuite, reason string, err error) {
+	msg := ""
+	if hErr, ok := humanerr.GetHumanReadableError(err); ok {
+		msg = hErr.Message
+	}
+
+	r.statusService.SetSuiteCondition(&suite.Status, testingv1alpha1.SuiteError, reason, msg)
+	r.Client.Status().Update(ctx, suite)
```

Comment:
error not handled, consider logging the error

Reply:
added multierror